definitions.yaml

```diff
@@ -0,0 +1,235 @@
+swagger: "2.0"
+info:
+  description: "This is an [Express](http://expressjs.com/) web application which implements some tasks to automate the provisioning of users in the Digital Citizenship Azure API management _developer portal_.\n\nThe aim is to automate some operations that would otherwise require the manual intervention of the APIs administrator: this lets developers start testing the API just after signing-up.\n\nUser's authentication takes place against an [Azure Active Directory B2C](https://azure.microsoft.com/en-us/services/active-directory-b2c/) tenant."
+  version: "0.1.0"
+  title: Digital citizenship portal
+
+paths: {}
+
+definitions:
+
+  #Returned by GET /subscriptions
+  #Returned by GET /subscriptions/:email
+  SubscriptionCollection: #TODO: add `nextLink` property?
+    type: array
+    items:
+      $ref: "#/definitions/SubscriptionContract"
+
+  #Returned by POST /subscriptions
+  #Returned by POST /subscriptions/:email
+  #Returned by PUT /subscriptions/:subscriptionId/:keyType
+  SubscriptionContract:
+    type: object
```

Comment:
remove this line, [this is how it should be](https://github.com/teamdigitale/io-utils/blob/master/src/__tests__/api.yaml#L49)